chemprop/nn/loss.py

```diff
@@ -81,6 +85,8 @@ def _calc_unreduced_loss(self, preds, targets, mask, weights, lt_mask, gt_mask)
         """Calculate a tensor of shape `b x t` containing the unreduced loss values."""
 
     def extra_repr(self) -> str:
+        if self.task_weights == torch.ones(1).view(1, -1):
+            return ""
         return f"task_weights={self.task_weights.tolist()}"
```

Comment:
```suggestion
        return f"task_weights={self.task_weights.tolist()}"
```

It's fine to display defaults. See pytorch defaults:
```python
>>> nn.Dropout()
Dropout(p=0.5, inplace=False)
```